metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -229,13 +229,20 @@ def _convert_value(param: parameters.Parameter):
         if kwargs.get(param.name) is not None
     }
 
+    # Add additional tags
+    if kfp_namespace:
+        tags = tags + (f"kfp_namespace:{kfp_namespace}",)
```

Comment:
Why add kfp namespace as a tag?  It's sent to the metadata service.

Reply:
Or it should be sent sent to the metadata service.

Reply:
same as reply below. would be good idea that to always by default tag the run by the experiment and namespace